### PR TITLE
Improve packaging

### PR DESCRIPTION
**Pull request by enoodle**
* Adding Tox for faster multi-runtime testing
* Adding PBR for automatic versioning
* Auto publishing to pypi

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Erez Freiberger <erez@epsagon.com>
```
Comment (ranrib) [inline AUTHORS]: Use dev@epsagon.com and support@epsagon.com in here

File: requirements.txt [hunk 2]
```diff
@@ -4,3 +4,4 @@ boto3==1.7.35
 simplejson==3.13.2
 six==1.11.0
 future==0.16.0
+Flask
```
Comment (ranrib) [inline requirements.txt]: It's a Dev requirement, not a prod one (whoever not using flask won't need that package)

**Comment by ranrib** [comment]
Looks perfect I love it!

File: AUTHORS (hunk 1 above)
Comment (enoodle) [reply AUTHORS]: it is autogenerated by PBR by the commits

Comment (ranrib) [reply AUTHORS]: 👍

**Pull request merged**